server/src/internalClusterTest/java/org/elasticsearch/indices/template/SimpleIndexTemplateIT.java

```diff
@@ -715,13 +716,15 @@ public void testStrictAliasParsingInIndicesCreatedViaTemplates() throws Exceptio
             .addAlias(new Alias("alias4").filter(termQuery("field", "value")))
             .get();
 
-        prepareIndex("a1").setId("test").setSource("{}", XContentType.JSON).get();
-        BulkResponse response = client().prepareBulk().add(new IndexRequest("a2").id("test").source("{}", XContentType.JSON)).get();
-        assertThat(response.hasFailures(), is(false));
-        assertThat(response.getItems()[0].isFailed(), equalTo(false));
-        assertThat(response.getItems()[0].getIndex(), equalTo("a2"));
-        assertThat(response.getItems()[0].getId(), equalTo("test"));
-        assertThat(response.getItems()[0].getVersion(), equalTo(1L));
+        client().prepareIndex("a1").setId("test").setSource("{}", XContentType.JSON).get();
```

Comment:
Does adding `client().` here change much? I haven't noticed a similar change elsewhere yet.

Reply:
This is probably the result of a bad merge by me. At some point recently someone removed all of these client() calls. I'll remove it.